### ISSUE TITLE
[Feature Request] 0-line for +ve and -ve display

### ISSUE THREAD

**Issue by trankillity** [kalkih/mini-graph-card#483]
There is no option to render a fixed line in this card. This would be very useful for target parameters, or for enabling display of the 0-line.

Currently the graph scales based on the lowest value, which means that negative values are shown in positive Y axis, rather than illustrating this with a line to show the 0-point. This is obviously undesirable, especially for a bar graph.

My use case is a ROI graph for my solar generation based on tariffs vs usage. Some cloudy days are negative ROI, so would like to see that illustrated.

Example [here](https://peltiertech.com/wp-content/img200806/InvertChart03.png).

**Comment by JJFourie** [comment]
This would indeed be very useful and appreciated. 
As the graph currently does not have any grid lines, it is not all that easy to orient the line with respect to reference values. 
My use case is when displaying temperatures, I'd like to show the freezing point ( 0°C ) as a reference. Using the existing functionality to change the line color at a specific threshold is not that useful if the full graph is still above/below the threshold (there will then be no different colors).
In addition, it would be nice to (optionally) also make it possible to set a "legend" for the line.

So to summarize, make it possible to configure:
1) Target/reference/zero lines, a line for each value provided.
2) Display the line description if configured.
3) Nice to have: make it possible to configure the line appearance
  - line color
  - thickness
  - dashed/solid

e.g.
![RainCard-Light](https://user-images.githubusercontent.com/49686555/104123748-3dd6b300-534d-11eb-9a8a-cc68bfc2ee76.jpg)

**Comment by RomRider** [comment]
Duplicate of #402
